app/portainer/services/authentication.js

```diff
@@ -96,19 +108,23 @@ angular.module('portainer.app').factory('Authentication', [
     }
 
     async function setUser(jwt) {
-      LocalStorage.storeJWT(jwt);
       var tokenPayload = jwtHelper.decodeToken(jwt);
       user.username = tokenPayload.username;
       user.ID = tokenPayload.id;
       user.role = tokenPayload.role;
       await setUserTheme();
     }
 
-    function isAdmin() {
-      if (user.role === 1) {
-        return true;
+    function tryAutoLoginExtension() {
+      if (!window.ddExtension) {
+        return false;
       }
-      return false;
+      console.debug('Auto-login Docker Desktop');
+      return login('admin', 'Passw0rd;');
```

Comment:
should this be env vars?

Reply:
Yes, this should not be hardcoded

Reply:
agree

Reply:
moved to constants for now, can be moved to env vars when it will need change

Reply:
resolving as PO are fine with hard coded for now with restriction to local host